new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js

```diff
@@ -21,7 +21,7 @@ const Cashbox = ({
   labelClassName,
   inFiatBalanceAlerts
 }) => {
-  const classes = cashboxClasses({ percent, cashOut })
+  const classes = cashboxClasses({ percent, cashOut, inFiatBalanceAlerts })
```

Comment:
I'm not really a fan of having a flag for a specific screen, as we might use that same behavior in the future, and the variable name shouldn't be tied to a specific screen. My suggestion is to call this flag something like `shouldApplyColorVariance`/`shouldApplyColorGradient` (I don't like the gradient one as much, as we're not working with gradients)

Reply:
Also, default value should be `false` when this flag is not specified in props. You can make the default value `true`, but the flag would need to be called something as `dontApplyColorVariance`

Reply:
Ah, noticed you're calling a pre-existing flag here. I guess you can ignore the variable name change